gnovm/cmd/gno/test.go

```diff
@@ -104,6 +105,13 @@ func (c *testCfg) RegisterFlags(fs *flag.FlagSet) {
 		"verbose output when running",
 	)
 
+	fs.BoolVar(
+		&c.failfast,
+		"failfast",
+		false,
+		"disables running additional tests after any test fails",
```

Comment:
```suggestion
		"do not start new tests after the first test failure",
```

to match golang's desc.